Mu2eUtilities/inc/TrackerBFieldInfo.hh

```diff
@@ -0,0 +1,44 @@
+#ifndef Mu2eUtilities_TrackerBFieldInfo_hh
+#define Mu2eUtilities_TrackerBFieldInfo_hh
+//
+// Find the magnetic field map that contains the tracker origin.
+// Return the map on request.
+//
+// I made this a class so that we can add other accessors that
+// transform the information received from the map.
+//
+// If a map cannot be located it throws an exception.
+// If there is more than one map among the BField innerMaps, it
+// also throws and exception.
+//
+// To use this to look at limits of a map:
+// TrackerBFieldInfo info
+//  cout << info.xmin() << " " << info.xmax() << endl;  // and so on
```

Comment:
I am confused: the comment speaks of an 'xmin' function in TrackerBFieldInfo, but I don't see that defined or implemented in this PR.  Did you perhaps forget to push recent changes?

Reply:
oops - I replied in the general comment box

Reply:
I updated the comments.